### PR TITLE
UG/ names are case-insensitive and disallow duplicate

### LINKED ISSUE #212: [PE-D] Adding Duplicate Names
Currently the add_student functionality will check duplication based on student name. Wondering if what happens if the tutor unfortunately has a student with the same name but different personal details

 

-------------
Labels: `severity.Low` `type.FeatureFlaw`
original: CSjiade/ped#2

### PR DESCRIPTION
**Pull request by nowknowing**
closes #212 #209

### CODE REVIEW

**Review by enhao25 — APPROVED** [review]
LGTM

File: docs/UserGuide.md [hunk 1]
```diff
@@ -254,6 +261,8 @@ Format: `add_student n/NAME p/STUDENT_PHONE_NUMBER e/EMAIL a/ADDRESS l/STUDY_LEV
 :bulb: TutorBuddy does not restrict the <code>STUDY_LEVEL</code> due to the vast number of education levels. The user 
 is free to input any education level.<br>
 Example: Any of the inputs here will be accepted. <code>Sec 2</code>, <code>Secondary 2</code>, <code>JC 1</code>, <code>Poly Y2</code>
+:bulb: If you have different students of the same <code>NAME</code> consider changing the name. <br>
+Example: Add a <code>John Doe Bespectacled</code> and a <code>John Doe Unadorned</code>
```
Comment (JonahhGohh) [inline docs/UserGuide.md:265]: I think you can use a page break here.
Current implementation:
image

add a br tag between line 263 and 264

**Review by JonahhGohh — CHANGES_REQUESTED** [review]
Need to add a br tag but otherwise looks good

**Review by JonahhGohh — APPROVED** [review]
LGTM

**Pull request merged**